packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -681,6 +686,8 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         const chunks = await tryFetchBlob<[string, string[]][]>(chunksBlobName) ?? [];
         const metadata = await tryFetchBlob<IContainerRuntimeMetadata>(metadataBlobName);
         const electedSummarizerData = await tryFetchBlob<ISerializedElection>(electedSummarizerBlobName);
+        const aliases = await tryFetchBlob<[string, string][]>(aliasBlobName) ?? [];
```

Comment:
I'm not sure how much real value this will provide (as most drivers are likely to either prefetch or bring all blobs together), but for inclusiveness of platform we might want to do parallelism here:

...
const electedSummarizerDataP = tryFetchBlob ...
const aliases = tryFetchBlobP ...

const [..., electedSummarizerData, tryFetchBlob] = await Promise.all([..., tryFetchBlobP, tryFetchBlobP]);

Reply:
that's a good idea